Source/ThirdParty/libwebrtc/Source/webrtc/api/video_codecs/video_encoder.cc

```diff
@@ -58,6 +58,23 @@ VideoCodecH264 VideoEncoder::GetDefaultH264Settings() {
   return h264_settings;
 }
 
+#ifdef WEBRTC_USE_H265
+VideoCodecH265 VideoEncoder::GetDefaultH265Settings() {
```

Comment:
this function is returning a complete object, I am not sure if this is good programming practice ...

Reply:
Ditto.